spec/validators/fee/warrant_fee_validator_spec.rb

```diff
@@ -8,60 +8,20 @@
   end
 
   include_examples 'common LGFS amount validations'
+  include_examples 'common warrant fee validations'
 
   describe '#validate_warrant_issued_date' do
     it 'should be valid if present and issued at least 3 months ago' do
       fee.warrant_issued_date = 3.months.ago
       expect(fee).to be_valid
     end
-
-    it 'should be invalid if present and too far in the past' do
-      fee.warrant_issued_date = 11.years.ago
-      expect(fee).to_not be_valid
-      expect(fee.errors[:warrant_issued_date]).to include 'check_not_too_far_in_past'
-    end
-
-    it 'should be invalid if present and in the future' do
-      fee.warrant_issued_date = 3.days.from_now
-      expect(fee).not_to be_valid
-      expect(fee.errors[:warrant_issued_date]).to include 'check_not_in_future'
-    end
-
-    it 'should be invalid if not present' do
-      fee.warrant_issued_date = nil
-      expect(fee).not_to be_valid
-      expect(fee.errors[:warrant_issued_date]).to eq( [ 'blank' ] )
-    end
   end
 
   describe '#validate_warrant_executed_date' do
-
-    it 'should raise error if before warrant_issued_date' do
-      fee.warrant_executed_date = fee.warrant_issued_date - 1.day
-      expect(fee).not_to be_valid
-      expect(fee.errors[:warrant_executed_date]).to eq( [ 'warrant_executed_before_issued'] )
-    end
-
     it 'should be invalid if present and too far in the past' do
       fee.warrant_executed_date = 11.years.ago
       expect(fee).to_not be_valid
       expect(fee.errors[:warrant_executed_date]).to include 'check_not_too_far_in_past'
     end
```

Comment:
why is this not shareable? -  i don't know

Reply:
@jsugarman I don't think it can be shared because the validations defined for warrant_executed_date are different in interim_fee_validator.rb (https://github.com/ministryofjustice/Claim-for-Crown-Court-Defence/blob/master/app/validators/fee/interim_fee_validator.rb) and warrant_fee_validator.rb (https://github.com/ministryofjustice/Claim-for-Crown-Court-Defence/blob/master/app/validators/fee/warrant_fee_validator.rb) specifically https://github.com/ministryofjustice/Claim-for-Crown-Court-Defence/blob/e3c7fa094fa0a3a0c9450ab677583ad64ea3b64e/app/validators/fee/warrant_fee_validator.rb#L16

Reply:
it probably is applicable as a general rule, but not a blocker